src/LexicographicHelpers.dfy

```diff
@@ -0,0 +1,246 @@
+// RUN: %dafny /compile:0 "%s"
```

Comment:
I'd like to leave this file out of this PR for now too. It's more relevant since it's providing a useful instance of ordering, but we can add that later.